swan-cern/files/swan_computing_config.py

```diff
@@ -137,7 +137,9 @@ async def _init_hadoop_secret(self):
         Create secret for Spark/Hadoop
         """
 
-        cluster = self.spawner.user_options[self.spawner.spark_cluster_field]
+        cluster = 'none'
```

Comment:
This is usually done in a more pythonic way in a one-liner.

Reply:
```suggestion
        cluster = self.spawner.user_options[self.spawner.spark_cluster_field] if self.spawner.user_options[self.spawner.software_source] == self.spawner.lcg_special_type else 'none'
```

Reply:
Or what if we just do `cluster = self.spawner.user_options.get(self.spawner.spark_cluster_field, 'none')`

Reply:
Isn't that worse for readability?